src/api/mappers/characterMapper.ts

```diff
@@ -0,0 +1,79 @@
+import { CharacterVoicedDto, CharacterInstanceDto, CharacterDto } from '../dtos/characterDto';
+import { CharacterNovel, Character, CharacterTrait, CharacterInstance, CharacterVoiced } from '../../models/character';
+import { imageFlaggingFromDto } from './imageFlaggingMapper';
+
+/**
+ * Maps dto into model.
+ * @param dto Dto.
+ * @returns Model.
+ */
```

Comment:
I think it's better to omit @returns and @param jsdoc tags to reduce docs duplication or make docs more meaningful.